docs/proof-of-stake/how-to-stake.md

```diff
@@ -23,6 +23,11 @@ A list of Voting Service Providers (VSPs) and statistics is maintained on the
 
 Using a VSP **does not give the VSP access to your funds**. All you are doing is granting voting rights to the VSP.
 
+!!! warning "Warning"
```

Comment:
Seeing as this warning is duplicated on two pages (and may be used on other pages in future) I recommend adding this warning to `mkdocs.yml`, similar to `seedWarning1`, `seedWarning2` etc. See `decrediton-setup.md` for an example usage.

Reply:
We should also use this new warning to replace the one in `using-decrediton.md`

Reply:
Added the warning as scriptWarning1 and scriptWarning2